Core/Util/GameManager.cpp

```diff
@@ -112,6 +112,14 @@ bool GameManager::Uninstall(std::string name) {
 }
 
 void GameManager::Update() {
+#ifdef HAVE_LIBNX
```

Comment:
There's every chance this may all be working now without these hacks - it should now join automatically after the install completes.  I think none of the changes in this file are needed anymore, hopefully.

-[Unknown]